api/src/main/java/org/ihtsdo/buildcloud/controller/BuildController.java

```diff
@@ -216,6 +228,7 @@ public void getPackageInputFile(@PathVariable final String releaseCenterKey, @Pa
 	}
 
 	@RequestMapping(value = "/{buildId}/outputfiles", method = RequestMethod.GET)
+	@IsAuthenticatedAsAdminOrReleaseManagerOrUser
```

Comment:
I think authors should be able to view everything but not do or change anything.

Reply:
Off course, the AUTHOR can view but can not edit. 
https://github.com/IHTSDO/snomed-release-service/blob/RBAC/data-service/src/main/java/org/ihtsdo/buildcloud/service/PermissionService.java#L44
When the user has the role of either AUTHOR or USER, then he can view but not edit.

Reply:
I misread the annotation, there are three roles mentioned, not just two. Okay, thanks.